File: packages/sdk-ts/src/manifests/zoom.ts
import { ToolManifest, Permission } from '../manifest.js';

export const zoomManifest = new ToolManifest({
  connector: 'zoom',
  description: 'Zoom Video Communications API',
  tools: {
    create_meeting: Permission.WRITE,
    get_recording: Permission.READ,
    cancel_meeting: Permission.DELETE,
    get_attendance_report: Permission.READ,
    add_panelist: Permission.WRITE,
    get_transcript: Permission.READ,
  },
});

File: packages/sdk-py/src/grantex/manifests/zoom.py
from grantex.manifest import ToolManifest, Permission

manifest = ToolManifest(
    connector="zoom",
    description="Zoom Video Communications API",
    tools={
        "create_meeting": Permission.WRITE,
        "get_recording": Permission.READ,
        "cancel_meeting": Permission.DELETE,
        "get_attendance_report": Permission.READ,
        "add_panelist": Permission.WRITE,
        "get_transcript": Permission.READ,
    },
)
